c_glib/arrow-glib/array.cpp

```diff
@@ -1050,6 +1176,25 @@ garrow_uint64_array_get_value(GArrowUInt64Array *array,
   return static_cast<arrow::UInt64Array *>(arrow_array.get())->Value(i);
 }
 
+/**
+ * garrow_uint64_array_get_values:
+ * @array: A #GArrowUInt64Array.
+ * @length: (out): The number of values.
+ *
+ * Returns: (array length=length): The raw values.
+ */
+const guint64 *
+garrow_uint64_array_get_values(GArrowUInt64Array *array,
+                               gint64 *length)
+{
+  auto arrow_array = garrow_array_get_raw(GARROW_ARRAY(array));
+  auto arrow_uint64_array =
+    std::static_pointer_cast<arrow::UInt64Array>(arrow_array);
+  *length = arrow_uint64_array->length();
+  return reinterpret_cast<const guint64 *>(arrow_uint64_array->raw_data());
```

Comment:
same question as above. I guess maybe it's just that they're declared in terms of platform C types?